File: athrill-account-client/projects/at-models/src/lib/system-values/at-entity.model.ts
import { BaseModel } from '../at-base.model';


export class ATEntityModel extends BaseModel
{
    public  EntityId : number;
    public  NameKey : string;
    public  DisplayName : string;
    public  Description : string;
    public  IsShownInRecordLogForAdminPanel : boolean;
    public  Id : number;
}

File: athrill-account-server/Libraries/Model/SystemValues/ATEntities/ATEntityModel.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Text;

namespace AT.Model.SystemValues.ATEntities
{
    public class ATEntityModel : BaseModel
    {
        public int EntityId { get; set; }
        [JsonIgnore]
        public string NameKey { get; set; }
        public string DisplayName { get; set; }
        [JsonIgnore]
        public string Description { get; set; }
        [JsonIgnore]
        public bool IsShownInRecordLogForSupportSite { get; set; }

        #region Override
        public override int Id => EntityId;
        #endregion
    }
}
